src/routes/index.ts

```diff
@@ -1,13 +1,15 @@
 import { Request, Response, Router } from 'express';
 import { responseSuccess } from '../utils/response.utils';
 import userRoutes from './UserRoutes';
+import productRoutes from './ProductRoutes';
 
 const router = Router();
 
-router.get('/api/v1/status', (req: Request, res: Response) => {
+router.get('/', (req: Request, res: Response) => {
   return responseSuccess(res, 200, 'This is a testing route.');
 });
 
 router.use('/user', userRoutes);
+router.use('/vendor/product', productRoutes);
 
```

Comment:
I propose that you can change this line to be
`product` instead of `/vendor/product` because both buyer and vendor will be accessing this routes  we are using hasRole for checking the role of user calling the route:
![Uploading image.png…]()

Reply:
@elijahladdie, the `hasRole` middleware ensures that only authorized users can access a particular service, About that route since vendors and buyers play different role on product, we need to separate path to these services each user getting what they need without confusion.

Reply:
Done.